test/d2l-filter-dropdown/d2l-filter-dropdown.js

```diff
@@ -1,387 +1,155 @@
-import '@polymer/iron-test-helpers/mock-interactions.js';
 import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
 
 (function() {
 	var filter;
```

Comment:
nit: this should be scoped to the individual `suite`. Not a problem currently but could cause a nasty bug. Same with `categories`.